python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md

```diff
@@ -51,26 +81,44 @@ python paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py
 ```
 The converted data binary file is saved by default in `~/.cache/paddle/dataset/int8/download/int8_full_val.bin`
  * ### Prepare model
-You can run the following commands to download ResNet50 model.
+You can run the following commands to download ResNet50 model. Since for performance optimization, we should be aware of the graph optimization during the QAT training phase, so we have the different fake QAT models between QAT MKL-DNN 1.0 and QAT MKL-DNN 2.0.
 
 ```bash
 mkdir -p /PATH/TO/DOWNLOAD/MODEL/
 cd /PATH/TO/DOWNLOAD/MODEL/
 export MODEL_NAME=ResNet50
+# QAT 1.0
 wget http://paddle-inference-dist.bj.bcebos.com/int8/QAT_models/${MODEL_NAME}_qat_model.tar.gz
+# QAT 2.0
+wget http://paddle-inference-dist.bj.bcebos.com/int8/QAT_models/${MODEL_NAME}_qat_perf.tar.gz
```

Comment:
Since due to naming differences this script will always fail to download one of the models I'd suggest the following:
```bash
mkdir -p /PATH/TO/DOWNLOAD/MODEL/
cd /PATH/TO/DOWNLOAD/MODEL/
# uncomment for QAT 1.0
# export MODEL_NAME=ResNet50
# export MODEL_FILE_NAME= ${MODEL_NAME}_qat_model.tar.gz
# uncomment for QAT 2.0
# export MODEL_NAME=resnet50
# export MODEL_FILE_NAME=${MODEL_NAME}_qat_perf.tar.gz
wget http://paddle-inference-dist.bj.bcebos.com/int8/QAT_models/${MODEL_FILE_NAME}
```
or something similar.

Reply:
Done